hibernate-cache/spring-local/src/main/resources/application.properties

```diff
@@ -0,0 +1,21 @@
+# Enable those entities marked with @Cacheable to be cached
+spring.jpa.properties.javax.persistence.sharedCache.mode=ENABLE_SELECTIVE
+# Enables second level cache
+spring.jpa.properties.hibernate.cache.use_second_level_cache=true
+# Enable query cache
+spring.jpa.properties.hibernate.cache.use_query_cache=true
+
+# Use Infinispan second level cache provider
+spring.jpa.properties.hibernate.cache.region.factory_class=infinispan
```

Comment:
I'm pretty sure some of those are set by default.

Reply:
None of those are default.